peers/app_request_network.go

```diff
@@ -165,6 +173,9 @@ func (n *appRequestNetwork) ConnectPeers(nodeIDs set.Set[ids.NodeID]) set.Set[id
 		return nil
 	}
 
+	// Add manually tracked peers
+	peers = append(peers, n.manuallyTrackedPeers...)
```

Comment:
I assume the system will behave correctly even if a peer appears more than once in the peers list (i.e., both returned by infoAPI.peers and is a member of the manuallyTrackedPeers).

Reply:
yes it will since we are using sets below and `ManuallyTrack` is idempotent.